tests/server/test_server.py

```diff
@@ -69,6 +74,17 @@ def execute_side_effect(self, query: str, params: list | tuple) -> None:
                     old_user = self._db_users[index]
                     # noinspection PyTypeChecker
                     self._db_users[index] = old_user[:3] + (old_user[3] - params[0],) + old_user[4:]
+            case QUERIES.WALLET_BUY_ADD_BTC:
+                try:
+                    index = [user[0] for user in self._db_users].index(params[1])
```

Comment:
```suggestion
                    index = [user[0] for user in self.db_users].index(params[1])
```
After rebase to master, all uses of private fields here must be replaced with properties

Reply:
Made changes to when _db_prices, _db_tokens are used.
_As far as I understand, because of the way db_user returns value, in case of .appends it should still use _db_user.